statistics.py

```diff
@@ -41,20 +70,11 @@ def obtain_statistics(cluster, endpoint, headers, host, port, stats_path):
         return
 
     consumed = maximum - free
-    timestamp = str(datetime.strftime(datetime.now(), '%s'))
-    out_string = f"{stats_path}.{cluster}.count {maximum} {timestamp}\n" \
-                 f"{stats_path}.{cluster}.used {consumed} {timestamp}\n" \
-                 f"{stats_path}.{cluster}.percent {consumed * 100 / maximum} {timestamp}\n"
-
-    # open socket
-    try:
-        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
-            s.connect((host, port))
-            # write data
-            s.sendall(bytes(out_string, 'utf-8'))
-            # close socket
-    except Exception as ex:
-        logger.warning(f"Exception while sending stats occurred: {ex}\n\nskipped\n")
+    percent = consumed * 100 / maximum
```

Comment:
I think we had cases when slot_limit was set to zero, in such a case the proposed code would crash on division by zero. Can you please make the code more robust?